src/connection/types.ts

```diff
@@ -3,6 +3,10 @@ import { Connector } from '@web3-react/types'
 
 export enum ConnectionType {
   UNISWAP_WALLET = 'UNISWAP_WALLET',
+  /**
+   * @deprecated
+   */
```

Comment:
```suggestion
  /** @deprecated - Use {@link UNISWAP_WALLET} instead. */
```

I think `@link` is right but it may be a different decorator.

Reply:
links back to enum value as expected